fuzzers/qemu_launcher/src/fuzzer.rs

```diff
@@ -23,38 +26,102 @@ use libafl::{
     monitors::MultiMonitor,
     mutators::scheduled::{havoc_mutations, StdScheduledMutator},
     observers::{HitcountsMapObserver, TimeObserver, VariableMapObserver},
+    prelude::LlmpRestartingEventManager,
```

Comment:
Use `events` instead of `prelude` (or prelude for everything, but mixing them doesn't make sense)

Reply:
I’ll sort that. Just an inconsistency from copy pasting I think.

Reply:
Done.